enterprise/signals.py

```diff
@@ -424,3 +424,14 @@ def generate_default_orchestration_record_display_name(sender, instance, **kwarg
 
 if COURSE_UNENROLLMENT_COMPLETED is not None:
     COURSE_UNENROLLMENT_COMPLETED.connect(enterprise_unenrollment_receiver)
+
+
+@receiver(pre_save, sender=models.EnterpriseCustomer)
```

Comment:
Any reason we are using pre_save and not post save?

Reply:
The sugesstion was about changing `@receiver(pre_save, sender=models.EnterpriseCustomer)` to `@receiver(post_save, sender=models.EnterpriseCustomer)`. we can cover both of them in one signal I guess.

Reply:
@zamanafzal One benefit that I noticed for the `pre_save` signal is that we don't have to explicitly call the `save()` (which will lead to max recursion error in this case) and the `update()` method. It will take care of it by itself.